src/script/interpreter.h

```diff
@@ -118,12 +125,14 @@ class TransactionSignatureChecker : public BaseSignatureChecker
     const CTransaction* txTo;
     unsigned int nIn;
     const CAmount amount;
+    const CachedHashes* cachedHashes;
```

Comment:
I'm wondering why they added this here, and not in the CachingTransactionSignatureChecker object